PuzzleTimeObjective-C/PuzzleTime/GameViewController.m

```diff
@@ -0,0 +1,382 @@
+//
+//  GameViewController.m
+//  PuzzleTime
+//
+//  Created by Wei Shan on 1/11/16.
+//  Copyright © 2016 Wei Shan. All rights reserved.
+//
+
+#import "GameViewController.h"
+#import "PuzzleCell.h"
+#import "UIView+Snapshot.h"
+
+@interface GameViewController ()
+
+@end
+
+@implementation GameViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    [_collectionView registerNib:[UINib nibWithNibName:@"PuzzleCell" bundle:nil] forCellWithReuseIdentifier:@"Cell"];
```

Comment:
Generally we prefer to use `self.` syntax to access properties, rather than touching the `_` ivar directly. So something like `self.collectionView` would be better here. This is so that you are always using the getters and setters for properties in case there is custom logic.